### PR TITLE
Feature fund startup #58

### PR DESCRIPTION
**Pull request by tmclean15**
Frontend and backend for funding a startup. The fund page can be reached by clicking the "fund this startup" button on the project page. I had to retool the payment_method handler, and the way that I was handling stripe on the frontend. The backend is now listening for a stripe webhook for when a user sets up a payment method.

### CODE REVIEW

File: client/src/components/ChoosePaymentMethod.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+  const handleAddPayment = async (event) => {
+    await setupPaymentMethod({
+      successRedirect: window.location.href,
+      errorRedirect: window.location.href,
+    });
+  };
+
+  const handleChangeMethod = (event) => {
+    setChosenMethod(event.target.value);
```
Comment (bonnieli) [inline client/src/components/ChoosePaymentMethod.js]: store `event.target.value` in a var

File: server/app.py [hunk 2]
```diff
@@ -16,8 +16,17 @@
 migrate = Migrate(app, db)
 bcrypt = Bcrypt(app)
 jwt = JWTManager(app)
+
+# Stripe setup
 stripe.api_key = Config.STRIPE_SECRET_KEY
 
+""" stripe.WebhookEndpoint.create(
+    url="http://a42d3d4cffbc.ngrok.io/api/v1/payment/payment-method",
```
Comment (bonnieli) [inline server/app.py]: perhaps just store an environment variable for this incase we forget to comment it out every time :)

File: client/src/components/ChoosePaymentMethod.js (hunk 1 above)
Comment (tmclean15) [reply client/src/components/ChoosePaymentMethod.js]: Added

File: server/app.py (hunk 2 above)
Comment (tmclean15) [reply server/app.py]: Great idea, will do!

**Review by MalcolmAG — APPROVED** [review]
Looks good to me

**Pull request merged**